docs/framework/component-demos.md

```diff
@@ -0,0 +1,8 @@
+---
+menu: Component demo's
+---
+
+# Demo's
+
+You can find a demo for the layout and different components on
+graphcommerce.vercel.app/test
```

Comment:
Can we remove this page until we actually have something pretty/complete to show?

Reply:
Just discussed with Paul. For now we will not provide a link in the documenation, untill we have a clearer purpose for these pages.